pkg/api/ldap_debug.go

```diff
@@ -100,7 +100,7 @@ func (user *LDAPUserDTO) FetchOrgs() error {
 }
 
 // ReloadLDAPCfg reloads the LDAP configuration
-func (hs *HTTPServer) ReloadLDAPCfg() response.Response {
+func (hs *HTTPServer) ReloadLDAPCfg(c *models.ReqContext) response.Response {
```

Comment:
No exceptions to the rule. Just changing a handler type to match the common pattern.